pkg/utils/release/sbom.go

```diff
@@ -0,0 +1,72 @@
+package release
```

Comment:
Can you please explain what mean every type here? adding a short description to every type can be enough. 

Example: https://github.com/redhat-appstudio/release-service/blob/main/api/v1alpha1/releaseplan_types.go#L23-L38